src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -564,7 +564,7 @@ export default class DefaultAudioVideoController
     if (!audioStream || audioStream.getAudioTracks().length < 1) {
       throw new Error('could not acquire audio track');
     }
-
```

Comment:
Suggest moving this to after the following `awaits` to `replaceAudioTrack` to ensure that the realtime controller is only updated if the track is actually replaced:

```
this._realtimeController.realtimeSetLocalAudioInput(audioStream);
this.meetingSessionContext.activeAudioInput = audioStream;
```